### PR TITLE
feat: Added example for Ory Keto Cloud

### PR DESCRIPTION
**Pull request by gen1us2k**

### CODE REVIEW

File: oathkeeper/12_ory_cloud_keto/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+      - ./nginx/index.html:/app/html:ro
+    networks:
+      - intranet
+
+  oathkeeper:
+    image: oryd/oathkeeper:v0.38
+    depends_on:
+      - kratos
```
Comment (vinckr) [inline oathkeeper/12_ory_cloud_keto/docker-compose.yml]:
```suggestion
```

I think this was from the other example?

File: oathkeeper/12_ory_cloud_keto/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## Running Ory proxy
+
+```
+   ory proxy http://127.0.0.1:8080
+```
+
+## Configuring Ory Keto Cloud
+
+1. Edit `cli.sh` and put your personal access token and change the URL of your Ory instance
+1. Run it `bash cli.sh`
```
Comment (vinckr) [inline oathkeeper/12_ory_cloud_keto/README.md:28]: Everything works fine up until here, I am getting some errors with the request:

```json
{"error":{"code":404,"status":"Not Found","request":"34ca2b32-9fce-9c8d-8601-749605e9aab9","reason":"Unknown namespace with name services.","message":"The requested resource could not be found"}}
```
Do I need to create a namespace first?

**Review by vinckr — CHANGES_REQUESTED** [review]
Looking good, 
I had some problem with the request however, see below

File: oathkeeper/12_ory_cloud_keto/docker-compose.yml (hunk 1 above)
Comment (gen1us2k) [reply oathkeeper/12_ory_cloud_keto/docker-compose.yml]: Yep

File: oathkeeper/12_ory_cloud_keto/README.md (hunk 2 above)
Comment (gen1us2k) [reply oathkeeper/12_ory_cloud_keto/README.md:28]: Yep. I wrote cli.sh especially for this

Comment (gen1us2k) [reply oathkeeper/12_ory_cloud_keto/README.md:28]: I'll update it soon

**Review by vinckr — CHANGES_REQUESTED** [review]
hello fails on startup for me, not sure if I am lacking a prerequisite? 

`hello_1       | standard_init_linux.go:228: exec user process caused: no such file or directory`

i also have 500 internal server error on http://127.0.0.1:4000/ui/login and http://127.0.0.1:8080/hello. below are the logs for login. i followed the steps in readme and cli.sh worked correctly.
is is a misconfiguration on my part or something with the containers?

```
2022/06/28 19:27:29 [error] 31#31: *1 auth request unexpected status: 302 while sending to client, client: 172.18.0.1, server: , request: "GET /hello HTTP/1.1", host: "127.0.0.1:8080"

2022/06/28 19:30:21 [error] 31#31: *6 auth request unexpected status: 404 while sending to client, client: 172.18.0.1, server: , request: "GET /ui/login HTTP/1.1", host: "127.0.0.1:4000"

2022/06/28 19:30:22 [error] 31#31: *8 auth request unexpected status: 404 while sending to client, client: 172.18.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1:4000", referrer: "http://127.0.0.1:4000/ui/login"
```

**Comment by vinckr** [comment]
Sorry I did not find the time to complete this example, but I will reopen / work on a different related example soon 😉

**Pull request closed without merging**